extension/content/personas.css

```diff
@@ -90,6 +91,15 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover:active {
```

Comment:
Small nit-pick: Could you give each selector its own line (like lines 71-75 for example)? That would greatly enhance readability.